docs/contributor/concepts/error-output-format.md

```diff
@@ -0,0 +1,42 @@
+# Output Error Format
+
+## Description
+
+I would propose standard error format for any CLI command:
+
+```text
+kyma provision --credentials-path ~/Desktop/fs-binding.txt --plan aws --region northeurope
+
+Error:
+  failed to provision: User is unauthorized for this operation
+
+Description:
```

Comment:
In this example we have two error descriptions called `Error` and `Description`. IMO we could rename: `Error`->`Original error` or `External system error`, `Description` -> `Error`.